controllers/flowcollector_controller_iso_test.go

```diff
@@ -98,8 +98,7 @@ func flowCollectorIsoSpecs() {
 					ExcludeInterfaces:  []string{},
 					Privileged:         false,
 					KafkaBatchSize:     0,
-					EnablePktDrop:      pointer.Bool(false),
-					EnableDNSTracking:  pointer.Bool(false),
+					Features:           []flowslatest.AgentFeature{},
```

Comment:
my bad: we need this:
```suggestion
					Features:           nil,
```